mappings/net/minecraft/world/biome/Biome.mapping

```diff
@@ -154,3 +166,40 @@ CLASS net/minecraft/class_1959 net/minecraft/world/biome/Biome
 			ARG 2 weight
 			ARG 3 minGroupSize
 			ARG 4 maxGroupSize
+	CLASS class_4762 MixedNoisePoint
+		COMMENT Represents a point in a multi-dimensional cartesian plane. Mixed-noise
+		COMMENT biome generator picks the closest noise point from its selected point
+		COMMENT and choose the biome associated to that closest point. Another factor,
+		COMMENT rarity potential, favors larger differences in values instead, contrary
+		COMMENT to other point values.
+		FIELD field_22043 temperature F
+		FIELD field_22044 humidity F
+		FIELD field_22045 hilliness F
+		FIELD field_22046 style F
```

Comment:
`style`?

Reply:
Something like a style/theme. Similar theme is favored.

Reply:
What does it do?

Reply:
Dunno, actually all 5 names are arbitrary guesses from hardcoded values given in mojang code and we aren't sure if any of them is correct at all